### PR TITLE
Add Lyx

### PR DESCRIPTION
**Pull request by izahn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/lyx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+source:
+  url: https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-2.3.6.1.tar.gz
+  sha256: 6d6f5458ebaac644cdfa35114d029e9ec57b4d930268d6d40bd9795d5c7e7929
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  rpaths:
+    - lib/
```
Comment (chrisburr) [inline recipes/lyx/meta.yaml:16]: This shouldn't be needed
```suggestion
```

Comment (izahn) [reply recipes/lyx/meta.yaml:16]: Thanks, I guess I learned bad habits from conda_r_skeleton_helper...

File: recipes/lyx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+{% set name = "lyx" %}
+{% set version = "2.13.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-2.3.6.1.tar.gz
```
Comment (chrisburr) [inline recipes/lyx/meta.yaml:9]: The version doesn't match, this should probably be fixed above and the url be changed to:
```suggestion
  url: https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-{{ version }}.tar.gz
```

File: recipes/lyx/build.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+if [[ ${target_platform} == osx-* ]]
+then
+    
+    ./configure --prefix=$PREFIX --with-x=no --enable-qt5 --without-included-boost --without-included-mythes
+else
+    ./configure --prefix=$PREFIX --enable-qt5 --without-included-boost --without-included-mythes
+fi
+
+make
```
Comment (chrisburr) [inline recipes/lyx/build.sh:11]:
```suggestion
make -j$CPU_COUNT
```

**Pull request merged**